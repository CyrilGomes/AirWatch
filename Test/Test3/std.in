2
individual1@mail.com
123456
3
