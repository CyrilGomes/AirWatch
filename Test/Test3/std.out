
=========================
 Welcome to AirWatch :
=========================

1. Login
2. Register
3. Exit
> :

Register as a private individual
--------------------------------

Email :
Password :

=========================
 Welcome to AirWatch :
=========================

1. Login
2. Register
3. Exit
> :
